### PR TITLE
OutputPanel: skeleton -> loading

### PR DESCRIPTION
**Pull request by jepsar**

### CODE REVIEW

File: primefaces/src/main/java/org/primefaces/component/outputpanel/OutputPanelRenderer.java [hunk 1]
```diff
@@ -121,10 +121,11 @@ protected boolean isDeferredNecessary(FacesContext context, OutputPanel panel) {
         if (!panel.isDeferred()) {
             return false;
         }
-        if (panel.getValueExpression(OutputPanel.PropertyKeys.loaded.name()) != null) {
+        boolean isAjaxRequest = context.getPartialViewContext().isAjaxRequest();
+        if (isAjaxRequest && panel.getValueExpression(OutputPanel.PropertyKeys.loaded.name()) != null) {
```
Comment (jepsar) [inline primefaces/src/main/java/org/primefaces/component/outputpanel/OutputPanelRenderer.java]: @Rapster you mean like this?

**Review by melloware — APPROVED** [review]

File: primefaces/src/main/java/org/primefaces/component/outputpanel/OutputPanelRenderer.java (hunk 1 above)
Comment (Rapster) [reply primefaces/src/main/java/org/primefaces/component/outputpanel/OutputPanelRenderer.java]: Almost, what I had in mind was:
```java
    protected boolean isDeferredNecessary(FacesContext context, OutputPanel panel) {
        boolean deferred = !context.getPartialViewContext().isAjaxRequest() && panel.isDeferred();
        if (deferred && panel.getValueExpression(OutputPanel.PropertyKeys.loaded.name()) != null) {
            deferred = !panel.isLoaded();
        }
        return deferred;
    }
```

Comment (jepsar) [reply primefaces/src/main/java/org/primefaces/component/outputpanel/OutputPanelRenderer.java]: Am I missing something? How is this going to work with Ajax requests?

Comment (Rapster) [reply primefaces/src/main/java/org/primefaces/component/outputpanel/OutputPanelRenderer.java]: I'm guessing everyone missed that comment https://github.com/primefaces/primefaces/issues/7712#issuecomment-901944480 😋 Not really happy with that fix in the past, as I ended up with a bug (same as mentionned in the comment). I'll try something

Comment (jepsar) [reply primefaces/src/main/java/org/primefaces/component/outputpanel/OutputPanelRenderer.java]: I'll revert this bit for now to reduce this PR to just the facet rename

Comment (Rapster) [reply primefaces/src/main/java/org/primefaces/component/outputpanel/OutputPanelRenderer.java]: Sounds good

**Pull request merged**